NOTICE.txt

```diff
@@ -2552,40 +2377,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
----
-
-## react-native-safe-area
```

Comment:
I'm unsure why this is getting removed.

Reply:
this one I can confirm is no longer used, we are using `react-native-safe-area-context`